source/features/image-alternatives.js

```diff
@@ -0,0 +1,29 @@
+export default async () => {
+	const imgContainers = document.querySelectorAll('.AdaptiveMedia-photoContainer, .Gallery-media');
+
+	for (const imgContainer of imgContainers) {
+		// Exit if it already exists
+		if (imgContainer.querySelector('.refined-twitter_image-alt')) {
```

Comment:
Since you're adding `refined-twitter_image-alt_parent-container` class on [line 19] , maybe you could use `classList.contains` instead of `querySelector` here, which will be slightly faster :)

[line 19]: #diff-05e70076331809ae0f6a4c2567917129R19

Reply:
👍